### PR TITLE
Update digest

### PR DESCRIPTION
**Pull request by modassarrana89**
Items performed :
1. Updated hack/build.sh to fix digest issue 
2. Added skopeo  installation in build& publish workflow
3. Tested build & publish workflow

### CODE REVIEW

**Comment by modassarrana89** [comment]
@dmage  Please review

File: hack/build.sh [hunk 1]
```diff
@@ -108,23 +108,9 @@ yq eval -i '
...
-        then .Descriptor.digest
-        else .[] | select(.Descriptor.platform.architecture=="ppc64le" and .Descriptor.platform.os=="linux") | .Descriptor.digest
-        end')
-
-Z_DIGEST=$(docker manifest inspect --verbose ${REGISTRY}/${NAMESPACE}/quay-operator-bundle:${TAG} | \
-    jq -r 'if type=="object"
-        then .Descriptor.digest
-        else .[] | select(.Descriptor.platform.architecture=="s390x" and .Descriptor.platform.os=="linux") | .Descriptor.digest
-        end')
+AMD64_DIGEST=$(skopeo inspect --raw  docker://${REGISTRY}/${NAMESPACE}/quay-operator-bundle:3.9-unstable | jq -r '.manifests[0].digest')
```
Comment (dmage) [inline hack/build.sh]: Please replace `3.9-unstable` with `${TAG}`

Comment (dmage) [reply hack/build.sh]: Also you shouldn't rely on manifests order. It's better to use
```
jq -r '.manifests[] | select(.platform.architecture == "amd64" and .platform.os == "linux").digest'
```

Comment (modassarrana89) [reply hack/build.sh]: ok

**Comment by modassarrana89** [comment]
@dmage  Please review . I have updated the change

**Review by dmage — APPROVED** [review]

**Pull request merged**